### PR TITLE
Add and fix English translations

### PR DESCRIPTION
**Pull request by ptsteadman**

### CODE REVIEW

**Comment by oO0oO0oO0o0o00** [comment]
Well... We do appreciate the efforts but when translating something an important point is that the original meaning, whether represented by the words directly or expressed somehow by the using of words should be kept as-is, which cannot be achieved w/o an clear and complete understanding of the context and certain larger circumstances. It is not that the translation was wrong but lack of contextual understanding can be seen from changed words. Not willing to explain it here it is no easy work. Still I do believe that any respond to the thread shall be present. Btw I'd like to know where did you find this GitHub repository?

Also keep in mind that my reply does not reflect any "official" opinion of the project (and its owner and co-owners) and suggests not the reason of the ignorance: maybe the stuff members are simply too tired to read any English.

File: README.md [hunk 1]
```diff
@@ -13,16 +13,25 @@ This simple repository requires no contribution of code, so that you can partici
...
 **当然，你还要事先准备至少一张你的女装照。**
 
-**Of course, you need to prepare a cross-dressing photo of yours first.**
+**Of course, you need to prepare a photo of you dressed in women's clothing first.**
 
 ### 文件夹命名规则和文件标准
+### File and Folder Naming Rules and Filetype Standards
+
  - 不接受盗图和未授权转载他人图片
+ - Stolen pictures and unauthorized photos of other people are not accepted.
```
Comment (lilydjwg) [inline README.md]: Pirate photos. From other people.

File: README.md [hunk 2]
```diff
@@ -13,16 +13,25 @@ This simple repository requires no contribution of code, so that you can partici
...
  - 请尽量使用有意义的文件夹名字
+ - Please attempt to use meaningful folder names.
```
Comment (lilydjwg) [inline README.md]: try to use.

File: README.md [hunk 3]
```diff
@@ -13,16 +13,25 @@ This simple repository requires no contribution of code, so that you can partici
...
  - 可使用你的 GitHub ID 或者其他社交平台上非中文的 ID 来命名文件夹
+ - Use your GitHub ID or roman character social media IDs as folder names.
  - 尽量不要在文件夹名字中出现空格
+ - Please try to avoid spaces in folder names.
  - 影响排序的文件夹命名 (例如 ```AAAA.Folder```, ```0.Folder```, ```111Folder```) 会被 Close Pull Request
+ - Pull requests with folder names that attempt to influence ordering (ex: ```AAAA.Folder```, ```0.Folder```, ```111Folder```）will be closed. 
  - 如果需要在你的文件夹内创建描述文件,请尽量使用 [Markdown](https://en.wikipedia.org/wiki/Markdown) 语法来编写
+ - If you need to add a descriptive file in your folder, please compose it using [Markdown](https://en.wikipedia.org/wiki/Markdown).
```
Comment (lilydjwg) [inline README.md]: in Markdown.

File: README.md [hunk 4]
```diff
@@ -13,16 +13,25 @@ This simple repository requires no contribution of code, so that you can partici
...
  - 图片请尽量压缩大小, 并 [删除 EXIF 信息](https://github.com/komeiji-satori/Dress/blob/master/CONTRIBUTING.md), 图片质量过低的 Pull Request 会被 Close
+ - Please attempt to compress your images and [delete EXIF data](https://github.com/komeiji-satori/Dress/blob/master/CONTRIBUTING.md), pull requests with excessively large image files will be closed.
```
Comment (lilydjwg) [inline README.md]: Please try to reduce file sizes and delete EXIF data. Pull requests with images of too low quality will be closed.

**Comment by oO0oO0oO0o0o00** [comment]
哇依云大姐姐~ QwQ

**Comment by ptsteadman** [comment]
> Well... We do appreciate the efforts but when translating something an important point is that the original meaning, whether represented by the words directly or expressed somehow by the using of words should be kept as-is, which cannot be achieved w/o an clear and complete understanding of the context and certain larger circumstances. It is not that the translation was wrong but lack of contextual understanding can be seen from changed words. Not willing to explain it here it is no easy work. Still I do believe that any respond to the thread shall be present. Btw I'd like to know where did you find this GitHub repository?

I found it via my GitHub feed (I guess one of the people I'm following starred it), and was curious. I'm not trying to start drama, if you're referring to the difference between "cross-dressing" and "wearing women's clothing" my thought was that the Chinese seems to be closer to the idea of wearing women's clothing rather than the idea of cross-dressing. But if this is problematic the other (added rather than changed) translations of the instructions might still be useful. 

Anyway just practicing my GitHub and translation :)

**Comment by oO0oO0oO0o0o00** [comment]
Well... If those with write permission are interested in globalization of the project. I guess not.

Whatever. Thanks. Meeting you and having the opportunity of chatting here with you is already a joyful experience. From your reply is it suggested that you can speak Chinese? (Maybe this is not a good chatroom)

**Comment by coffeepowereddev** [comment]
Regarding "cross-dressing" vs "wearing womens' clothing": the former implies that the wearer is not female, but the contributing guidelines state that there is no restriction on gender.

**Review by BBleae — APPROVED** [review]

**Pull request merged**